### ISSUE TITLE
Use Kudos Sets to help source talent

### ISSUE THREAD

**Issue by PixelantDesign** [gitcoinco/web#4304]
## User Story
As a funder/recruiter looking for talent, I'd like a quick glance to understand what the Kudos in a person's profile equates to.

## Why
Sometimes it's hard to parse through all the Kudos on a user's profile.

## What
Idea 2: What if we create collectable sets of Kudos
that equal different certification on the Gitcoin platform? We make it easy for people finding workers to understand skillsets.

This format can be applied to many things....

<img width="1048" alt="Screen Shot 2019-04-29 at 9 05 46 AM" src="https://user-images.githubusercontent.com/238235/56903069-94ad3380-6a69-11e9-96c4-d81b73a6d2bc.png">

You’re a python developer…python developers on our platform also have these other badges….

## Revenue Opportunity
- Payment for access to this info

**Comment by owocki** [comment]
i love this idea!
